recipes/pytesseract/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pytesseract" %}
+{% set version = "0.2.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+  url: https://github.com/madmaze/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: b389cf1208df8567f953b8da5b716b28f4c42f149f367b4e51b73ef651a55707
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - pip
+
+  run:
+    - python
+    - flake8
+    - pep8-naming
+    - flake8-quotes
+    - pytest
+    - pillow 5.4.1
+
+test:
+  imports:
+    - PIL
+    - PIL.Image
+    - pytesseract
+
+
+about:
+  home: https://github.com/madmaze/pytesseract
+  license: GPL-3.0
+  license_family: GPL
+  summary: 'Python-tesseract is an optical character recognition (OCR) tool for python.'
+
+  description: |
+   Python-tesseract is a wrapper for Google's Tesseract-OCR Engine.
+   It is also useful as a stand-alone invocation script to tesseract,
+   as it can read all image types supported by the Pillow and Leptonica
+   imaging libraries, including jpeg, png, gif, bmp, tiff, and others.
+   Additionally, if used as a script, Python-tesseract will print the
+   recognized text instead of writing it to a file
+
+
+extra:
+  recipe-maintainers:
+    - adbeda
+    - agcopenhaver
```

Comment:
@agcopenhaver are you OK with being a maintainer?

Reply:
Sure thing @scopatz